api/app/signals/apps/api/v1/serializers.py

```diff
@@ -811,3 +813,32 @@ class Meta:
         fields = (
             'id',
         )
+
+
+class UserFilterSerializer(HALSerializer):
+    serializer_url_field = UserFilterLinksField
+    _display = DisplayField()
+
+    class Meta:
+        model = Filter
+        fields = (
+            '_links',
+            '_display',
+            'name',
+            'created_at',
+            'options',
+        )
+
+    def validate(self, attrs):
+        from signals.apps.api.v1.filters import SignalFilter
+        signal_filter = SignalFilter(data=attrs['options'], queryset=Signal.objects.none())
```

Comment:
nice! As usual, the documentation of `django-filter`'s `FilterSet` class does not mention these keyword arguments ... Same for `.is_valid()` below.